### DISCUSSION TITLE
How to run cron jobs with foundry?

### DISCUSSION THREAD

**Discussion by armatrix** [foundry-rs/foundry#8711] [Help]
For example, if I want to implement keeper logic, I can do it with setInterval() in hardhat, but in foundry I can only do it with node or some other language. Is there a more native way for me to do this?
For example, in foo.s.sol, I can create a child process like `vm.cron(funcA, 1 seconds)` to do this.

**Comment by amalnathsathyan** [comment]
Did you check`vm.warp()` and `vm.roll()` functions to adjust time interval ?

**Reply by zerosnacks** [reply]
I don't think this is something Foundry supports out-of-the-box but this could possibly be achieved by setting cron-jobs using `vm.ffi`
